snapshots/commands/connection_string.test.snap

```diff
@@ -1,70 +1,70 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
-exports[`connection_string connection_string branch id 8 digits test 1`] = `
-"postgresql://test_role:test_pwd@ep-lame-king-930914.us-east-2.aws.neon.tech/test_db?sslmode=require
+exports[`connection_string > connection_string > test 1`] = `
+"postgresql://test_role:test_pwd@ep-cool-king-930914.us-east-2.aws.neon.tech/test_db?sslmode=require
```

Comment:
I like that change.
Gives a feeling of progress.

Reply:
😅 

Seems due to the added `>` separator, tests are sorted a bit differently now.

Reply:
Hope you've noticed how "lame" changed to "cool" :)